CHANGELOG.rst

```diff
@@ -70,6 +70,7 @@ Protocol is now at version **1.16**. See `API changelog`_.
   bucket (#1137)
 - Removed Structlog binding and bottlenecks (fixes #603)
 - Fixed Swagger output with subpath and regex in pyramid routes (fixes #1180)
+- Fixed Postgresql errors when trying to compare numeric value with an empty string. (fixes #1194)
```

Comment:
For me the fix is at the storage level not the request level, but if it feels more natural, I can change the description.